src/layout.ts

```diff
@@ -44,24 +44,42 @@ export type LinkObject = {
   url: string;
 };
 
-export function layoutPage(content: Paragraph[], box: Box, fonts: Font[]): Frame {
+export function layoutPages(paragraphs: Paragraph[], box: Box, fonts: Font[]): Frame[] {
+  const pages = [];
+  let remainingParagraphs = paragraphs;
+  while (remainingParagraphs?.length) {
+    const { frame, remainder } = layoutPage(remainingParagraphs, box, fonts);
+    remainingParagraphs = remainder;
+    pages.push(frame);
+  }
+  return pages;
+}
+
+export function layoutPage(content: Paragraph[], box: Box, fonts: Font[]) {
```

Comment:
Just to proof I'm actually reading the change: I think it should be `contents` here.

Reply:
Good point. I've renamed to parameter to `paragraphs`, which also matches the type. Before I had a term for that, I used to just call this array the `content`.